crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -287,59 +287,59 @@ module sui::sui_system {
         )
     }
 
-    /// Add delegated stake to a validator's staking pool.
-    public entry fun request_add_delegation(
+    /// Add stake to a validator's staking pool.
+    public entry fun request_add_stake(
         wrapper: &mut SuiSystemState,
-        delegate_stake: Coin<SUI>,
+        stake: Coin<SUI>,
         validator_address: address,
         ctx: &mut TxContext,
     ) {
         let self = load_system_state_mut(wrapper);
-        validator_set::request_add_delegation(
+        validator_set::request_add_stake(
             &mut self.validators,
             validator_address,
-            coin::into_balance(delegate_stake),
+            coin::into_balance(stake),
             option::none(),
             ctx,
         );
     }
 
-    /// Add delegated stake to a validator's staking pool using multiple coins.
-    public entry fun request_add_delegation_mul_coin(
+    /// Add stake to a validator's staking pool using multiple coins.
+    public entry fun request_add_stake_mul_coin(
```

Comment:
We should really just remove this method.

Reply:
To keep things clean I might leave this in for now so this is a strict renaming pass, but will do a fast follow to remove this.

Reply:
But I believe this function is used in our `sui-transaction-builder` as well as `sui-rosetta`:
https://github.com/MystenLabs/sui/blob/ffc34df782a597e62e8191b934fa752863bdb376/crates/sui-transaction-builder/src/lib.rs#L663
cc @patrickkuo